src/main/java/computer_science_basics/homework_1/Task2.java

```diff
@@ -15,7 +15,10 @@ public static void main(String[] args) {
     }
 
     public static int findSingleNumber(int[] array) {
-        // TODO: Your code here
-        return 0;
+        int singleNumber = array[0];
+        for (int i = 1; i < array.length ; i++) {
+            singleNumber ^= array[i];
+        }
+        return singleNumber;
```

Comment:
Please revert this changes. 
You don't need to modify Task2 file
You have already created a new file